src/css/global.scss

```diff
@@ -76,6 +76,7 @@ html {
   -moz-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   scroll-padding-top: var(--header-height);
+  // scroll-behavior: smooth;
```

Comment:
tried adding this to scroll smoothly to `async-collaboration` but it interfered with the scrolly telling